store/actions/geocodingActions.js

```diff
@@ -0,0 +1,44 @@
+import {
+    GEOCODING_GET_REQUEST,
+    GEOCODING_GET_SUCCESS,
+    GEOCODING_GET_FAILURE
+} from '../types/geocodingActionTypes';
+import axios from 'axios';
+
+const apiKey = 'AIzaSyC1hiS3sXkvQPqEodYESWOLBYkUsyR8Tv4'; // Google Maps API key
+
+export const geocodingGetRequest = () => ({
+    type: GEOCODING_GET_REQUEST,
+});
+
+export const geocodingGetSuccess = (coordinates) => ({
```

Comment:
We nare not creating separate functions to handle action types. 
Check the other action files